ci.properties

```diff
@@ -1,4 +1,5 @@
 ci.pull.request.auto.comment=
+ci.pull.request.auto.comment[liferay-platform-experience]="Thank you for sending this PR to liferay-platform-experience. If you haven't already, take this opportunity to review our [pull request guidelines](https://liferay.atlassian.net/wiki/spaces/PEDS/pages/2915598365). For additional help, reach out via the appropriate channels listed [here](https://liferay.atlassian.net/wiki/spaces/PEDS/overview#%3Aslack%3A-Slack-Channels)."
```

Comment:
For additional help, reach out via the appropriate channels listed [here](https://liferay.atlassian.net/wiki/spaces/PEDS/overview#%3Aslack%3A-Slack-Channels)."

Might be better to update this like so:

For additional help, please reach out via the [appropriate channels](https://liferay.atlassian.net/wiki/spaces/PEDS/overview#%3Aslack%3A-Slack-Channels)."

This will make this more accessible since the link will have an understandable label, not just "here".

Reply:
I made it based on the existing comment from liferay-search, I wanted to keep it consistent so Brian doesn't reject it :sweat_smile: But I'll give it a try